### DISCUSSION TITLE
Puzzle 55 Error in the final Select

### DISCUSSION THREAD

**Discussion by 2b4fITin** [smpetersgithub/AdvancedSQLPuzzles#40] [General]
Seems in the `WHERE` clause for _Matches in both table A and table B_ we need to add `and QuantityA = QuantityB`?

**Comment by smpetersgithub** [comment]
Thank you!!  I will check on this.  Btw, I still haven't checked the other puzzle you possibly found an error with.  I will check them shortly.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: 2b4fITin ***@***.***>
Sent: Sunday, June 9, 2024 9:46:46 AM
To: smpetersgithub/AdvancedSQLPuzzles ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [smpetersgithub/AdvancedSQLPuzzles] Puzzle 55 Error in the final Select (Discussion #40)


Seems in the 'WHERE' clause we need to add and QuantityA = QuantityB

**Reply by 2b4fITin** [reply]
It's ok. Take your time. Thanks a lot!

**Comment by smpetersgithub** [comment]
Thank you again!  I also added `ProductNameA = ProductNameB` for the criteria _Quantities in table A and table B do not match_

I need to add more records to the test data in the puzzle to create multiple instances where the product name matches, but the quantities differ.

Please let me know if you find any more errors.
